madgraph5amcatnlo.spec

```diff
@@ -35,6 +38,19 @@ rm bin/compile.py
 # Save patched config
 mv input/mg5_configuration_patched.txt input/mg5_configuration.txt
 
+# Start small NLO event generation to make sure that all additional packages are compiled
+cat <<EOF > basiceventgeneration.txt
+generate p p > t t~ [QCD]
+output basiceventgeneration
+launch
+set nevents 5
+EOF
+./bin/mg5_aMC ./basiceventgeneration.txt
```

Comment:
as before - why do we need this - the package must build itself.

Reply:
Hello @davidlange6  ,

This is required for the installation of subpackages that MG5aMC@NLO uses.
I summarized the issue and asked in regards of the best implementation in the hypernews thread https://hypernews.cern.ch/HyperNews/CMS/get/softwareDistrib/757.html

Kind regards,
Andrej

Reply:
it seems we need a discussion of this topic to understand what is really happening.. (from your HN I only add to by confusion)